scrapy/http/request/form.py

```diff
@@ -10,21 +10,16 @@
 from typing import TYPE_CHECKING, Any, Iterable, List, Optional, Tuple, Union, cast
 from urllib.parse import urlencode, urljoin, urlsplit, urlunsplit
 
-from lxml.html import (
-    FormElement,
-    HTMLParser,
-    InputElement,
-    MultipleSelectOptions,
-    SelectElement,
-    TextareaElement,
-)
-from parsel.selector import create_root_node
+from lxml.html import FormElement  # nosec
```

Comment:
Question: We need to change import style because is not possible to use # no sec otherwise? Thanks

Reply:
isort did that automatically, not sure why, I imagine it is a style choice.

Reply:
Got it! Thanks!